examples/tutorial-standalone/main.tf

```diff
@@ -0,0 +1,198 @@
+  confidential_dataset_id     = "secured_dataset"
+  taxonomy_name               = "secured_taxonomy"
+  taxonomy_display_name       = "${local.taxonomy_name}-${random_id.suffix.hex}"
+  confidential_table_id       = "irs_990_ein_re_id"
+  non_confidential_table_id   = "irs_990_ein_de_id"
+  bq_schema_irs_990_ein       = "ein:STRING, name:STRING, ico:STRING, street:STRING, city:STRING, state:STRING, zip:STRING,group:STRING, subsection:STRING, affiliation:STRING, classification:STRING, ruling:STRING, deductibility:STRING, foundation:STRING, activity:STRING, organization:STRING, status:STRING, tax_period:STRING, asset_cd:STRING, income_cd:STRING, filing_req_cd:STRING, pf_filing_req_cd:STRING, acct_pd:STRING, asset_amt:STRING, income_amt:STRING, revenue_amt:STRING, ntee_cd:STRING, sort_name:STRING"
+}
+
+resource "random_id" "suffix" {
+  byte_length = 4
+}
+
+module "secured_data_warehouse" {
+  source = "../.."
+
+  org_id                           = var.org_id
+  data_governance_project_id       = module.base_projects.data_governance_project_id
+  confidential_data_project_id     = module.base_projects.confidential_data_project_id
+  non_confidential_data_project_id = module.base_projects.non_confidential_data_project_id
+  data_ingestion_project_id        = module.base_projects.data_ingestion_project_id
+  sdx_project_number               = module.template_project.sdx_project_number
+  terraform_service_account        = var.terraform_service_account
+  access_context_manager_policy_id = var.access_context_manager_policy_id
+  bucket_name                      = "data-ingestion"
+  pubsub_resource_location         = local.location
+  location                         = local.location
+  trusted_locations                = ["us-locations"]
+  dataset_id                       = local.non_confidential_dataset_id
+  confidential_dataset_id          = local.confidential_dataset_id
+  cmek_keyring_name                = "cmek_keyring"
+  delete_contents_on_destroy       = var.delete_contents_on_destroy
+  perimeter_additional_members     = var.perimeter_additional_members
+  data_engineer_group              = var.data_engineer_group
+  data_analyst_group               = var.data_analyst_group
+  security_analyst_group           = var.security_analyst_group
+  network_administrator_group      = var.network_administrator_group
+  security_administrator_group     = var.security_administrator_group
+
+  depends_on = [
+    module.base_projects,
+    module.iam_projects,
+    module.centralized_logging,
+    google_project_iam_binding.remove_owner_role
+  ]
+}
+
+module "de_identification_template" {
+  source = "../..//modules/de-identification-template"
+
+  project_id                = module.base_projects.data_governance_project_id
+  terraform_service_account = var.terraform_service_account
+  crypto_key                = module.tek_wrapping_key.keys[local.kek_key_name]
+  wrapped_key               = local.wrapped_key_secret_data
```

Comment:
shouldn't this come from the `harness` OR the secret mgr?

Reply:
The `local.wrapped_key_secret_data` is using the secret manager

```hcl
locals {
...
 wrapped_key_secret_data     = chomp(data.google_secret_manager_secret_version.wrapped_key.secret_data)
...
}
```
We will move the local from `harness.tf` to `main.tf` to keep the declaration and the usage in the same file.

Reply:
We moved the local from harness.tf to main.tf to keep the declaration and the usage in the same file.